### ISSUE TITLE
15.0-dev branch for payroll

### ISSUE THREAD

**Issue by ghost** [OCA/payroll#58]
I appreciate this active payroll community! For version 15.0 I would like to explore ways to make the payroll easier to use, both for small and big companies. Possible tools for this could be a 15.0-dev branch and Google document/spreadsheet/survey/meeting.

My PR #51 could be a contribution to a 15.0-dev branch. It should not be merged with 14.0 since it is adding a new payroll dependency on the **analytic** module. As discussed in #31 it is not good to add new dependencies for the current users of the module.

To avoid maintaining two branches more than necessary, we can wait with this for a couple of weeks, to finish important improvements for 14.0.

**Comment by nimarosa** [comment]
@appstogrow Hello Henrik, I agree that we should continue improving the module and also glad that people are starting to contribute again. 

About the new branch I have a different opinion: 

I think we should continue developing the new features or improvements in the version 14.0 for many reasons: 

- Version 14.0 and 15.0 are no so different in terms of migrating this module: this is because there are not major changes in odoo base core so when we are ready we can expect the migration to version 15.0 to happen quickly.

- I think that if we develop for 14.0 it will be a more clean and improved version the one migrated to 15.0. In terms of views and unoptimized code and functions the module has a lot of things to work and discuss yet. 

- I think we should not continue migrating a no optimized/finished module and today we have the chance to leave a v14.0 very functional and optimized, that way in the next versions of odoo we will have a very solid module. If we continue migrating we will have a very discontinued version set and the module will be very different between versions => this is bad for final users. 

- A lot of users continue using v14.0 and would not be benefited with the new features if we develop for 15.0. I think this will kill a bit the hype of new contributions and participation in the repo that we saw the last months. 

- This one is maybe more personal but I have a bunch of PRs prepared for 14.0 that I extracted from private modules to improve directly in this repo. If we move to 15.0 maybe not both of us, but I, will have to maintain the two branches anyways. 

On resume I think we should wait maybe a couple of months more to finish merging more features and improvements, we have issues open like the one to merge payroll_cancel and change_state that are pending to do, also more refactoring like discussed. 

There are a lot of OCA repos that sos not migrated to 15.0 yet so I think in terms of time we are okay. Remember that payroll is very tied with all hr modules so we should also wait for repo oca/hr and hr-holidays finish migration to 15.0. 

I propose we continue doing the new features and fixes in the v14.0 and then we will have a very solid module for v15.
